README.md

```diff
@@ -646,6 +646,8 @@ _Institutional repository and digital library software._
 
 ## Federated Identity/Authentication
 
+- [SimpleLogin](https://simplelogin.io) - Open source OAuth2/OpenID Connect Identity Provider that provides a privacy-first alternative to the "Sign in with Facebook|Google|Apple" button. ([Source Code](https://github.com/simple-login/app)) `MIT` `Python`
```

Comment:
I think the intention behind just having a link in this section was to forward all entries to that list.

Reply:
You're right, I will create another PR on this repo then.